sources/views/contacts/formContact.js

```diff
@@ -0,0 +1,206 @@
+																{
+																	view: "uploader",
+																	value: "Change photo",
+																	accept: "image/png, image/gif, image/jpg, image/jpeg",
+																	localId: "imageUploader",
+																	autosend: false,
+																	multiple: false
+																},
+																{
+																	view: "button",
+																	value: "Delete Photo",
+																	click: () => this.deletePhoto()
+																}
+															]
+														}
+
+													]
+												}
+											]
+										}
+									]
+								},
+								{},
+								{
+									cols: [
+										{},
+										{
+											view: "button",
+											value: "Cancel",
+											width: 150,
+											click: () => this.closeForm()
+										},
+										{
+											view: "button",
+											localId: "btnAdd",
+											value: "#nameButton#",
+											width: 150,
+											click: () => this.addNewContactOrUpdate()
+										}
+									]
+								}
+							]
+						}
+					],
+					rules: {
+						FirstName: webix.rules.isNotEmpty,
+						LastName: webix.rules.isNotEmpty,
+						StatusID: webix.rules.isNotEmpty,
+						Company: webix.rules.isNotEmpty,
+						Job: webix.rules.isNotEmpty
+					}
+				}
+			]
+		};
+	}
+
+	init() {
+		this.formComponent = this.$$("formForContactData");
+		this.image = this.$$("templateImage");
+		const reader = new FileReader();
+		this.$$("imageUploader").attachEvent("onBeforeFileAdd", (upload) => {
+			reader.readAsDataURL(upload.file);
+			reader.onload = () => this.image.setValues({Photo: reader.result});
+		});
+
+		const id = ItemDataContact.getId();
+		const nameForm = id ? "Edit" : "Add";
+		const nameButton = id ? "Save" : "Add";
+		this.$$("header").setValues({nameForm});
+		this.$$("btnAdd").setValue(nameButton);
+		if (id) {
+			let item = ItemDataContact.getItem();
+			this.formComponent.setValues(item);
+			this.image.setValues({Photo: item.Photo});
+		}
+	}
+
+	closeForm() {
+		this.formComponent.clear();
+		this.formComponent.clearValidation();
+		if (!ItemDataContact.getId()) {
+			const idFirstItem = dataContacts.getFirstId();
+			ItemDataContact.setId(idFirstItem);
+		}
+		this.show("/top/contacts/contacts.contactDescription");
+	}
+
+	addNewContactOrUpdate() {
+		if (this.formComponent.validate()) {
+			const idFromData = ItemDataContact.getId();
+			const dataFromForm = this.formComponent.getValues();
+			dataFromForm.Photo = this.image.getValues().Photo;
+			dataContacts.waitSave(() => {
+				if (idFromData) {
+					dataContacts.updateItem(dataFromForm.id, dataFromForm);
+					webix.message("Contact was updated");
+				}
+				else {
+					dataContacts.add(dataFromForm, 0);
+					webix.message("Contact was added");
+				}
+			}).then((res) => {
+				ItemDataContact.setId(res.id);
+				this.closeForm();
+			});
+		}
+		else {
+			webix.message("Please check fields");
+		}
+	}
+
+	deletePhoto() {
+		this.image.setValues({Photo: avatar});
```

Comment:
During data saving `Photo` is extracted from this template and saved to item.
As a result, `avatar` will be saved as a part of data (which is not right). 
Set empty value here and make sure the placeholder is still rendered correctly if contact info does not contain `Photo`.